### PR TITLE
Updates to prepare for publishing Config PSD01

### PR DESCRIPTION
**Pull request by jamsden**
Please review before I create the zip files and send out the notice of intent to publish to the PGB.

### CODE REVIEW

File: specs/config/config-shapes.ttl [hunk 1]
```diff
@@ -31,11 +31,11 @@
     dcterms:title            "Resource Shapes for OSLC Configuration Management" ;
     dcterms:description      "Shapes for resources defined by OSLC Configuration Management."^^rdf:XMLLiteral ;
     dcterms:publisher        <https://open-services.net/about/> ;
-    dcterms:source           <https://github.com/oslc-op/oslc-specs/blob/master/specs/config/config-shapes.ttl> ;
-    dcterms:isPartOf         <https://oslc-op.github.io/oslc-specs/specs/config/oslc-config-mgt.html> ;
-    dcterms:hasVersion       "WD" ;
+    dcterms:source           <https://docs.oasis-open-projects.org/oslc-op/config/v1.0/psd01/config-shapes.ttl> ;
+    dcterms:isPartOf         <https://docs.oasis-open-projects.org/oslc-op/config/v1.0/psd01/oslc-config-mgt-spec.html> ;
+    dcterms:hasVersion       "PWD" ;
```
Comment (berezovskyi) [inline specs/config/config-shapes.ttl:36]: Psd not pwd

File: specs/config/config-resources.html [hunk 2]
```diff
@@ -76,13 +76,16 @@
...
-            href: "config-resources.html",
+            title: "OSLC Configuration Management 1.0: Versioned Resources",
+            href: thisBase + "versioned-resources.html",
           },
-          { title: "OSLC Configuration Management 1.0 Part 4: RDF Vocabulary", href: "config-vocab.html" },
+          {
+            title: "OSLC Configuration Management 1.0: Configuration Specification (this document)",
+            href: thisBase + "config-resources.html",
+          },
+          { title: "OSLC Configuration Management 1.0: RDF Vocabulary", href: thisBase + "config-vocab.html" },
```
Comment (berezovskyi) [inline specs/config/config-resources.html:88]: Machine readable resources should be referenced too, without part numbers. Any reason part numbers were removed from the main parts of the spec?

File: specs/config/config-shapes.ttl [hunk 3]
```diff
@@ -31,11 +31,11 @@
...
     dcterms:license          <http://www.apache.org/licenses/LICENSE-2.0> ;
-    #dcterms:issued          "2020-xx-xx"^^<http://www.w3.org/2001/XMLSchema#date> ;
+    dcterms:issued          "2021-09-02"^^<http://www.w3.org/2001/XMLSchema#date> ;
     dcterms:dateCopyrighted  "2012-2020" .
```
Comment (berezovskyi) [inline specs/config/config-shapes.ttl:39]: 2021

File: specs/config/config-vocab.html [hunk 4]
```diff
@@ -16,7 +16,7 @@
       var oasisBase = "https://docs.oasis-open-projects.org/oslc-op/config/v1.0";
       var rev = "01";
       var thisBase = wdBase;
-      if (status != "WD") {
+      if (status != "PSD") {
```
Comment (berezovskyi) [inline specs/config/config-vocab.html:19]: should be wd; psd status should be set in respecConfig

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Thanks for the draft package! Please see the comments attached.

File: specs/config/config-resources.html (hunk 2 above)
Comment (jamsden) [reply specs/config/config-resources.html:88]: I will get that on the next PSD. No need to hold up initial publishing for that.

File: specs/config/config-vocab.html (hunk 4 above)
Comment (jamsden) [reply specs/config/config-vocab.html:19]: fixed

File: specs/config/config-shapes.ttl (hunk 1 above)
Comment (jamsden) [reply specs/config/config-shapes.ttl:36]: Fixed

File: specs/config/config-resources.html (hunk 2 above)
Comment (jamsden) [reply specs/config/config-resources.html:88]: I added them

Comment (jamsden) [reply specs/config/config-resources.html:88]: Also there was a missing slash in the generated path.

**Review by berezovskyi — APPROVED** [review]

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**